gulpfile.js

```diff
@@ -61,14 +61,6 @@ var TEST_SETS = [
 // Ignore ts-compile errors while watching (but not in normal builds).
 var watching = false;
 
-var TS_SETTINGS = {
-  sortOutput: true,
-  declarationFiles: true,
```

Comment:
Why did we get rid of settings for sortOutput, declarationFiles, and noExternalResolve?